docker-compose-configuration/metaphactory-docker-compose/rds-local/.env

```diff
@@ -0,0 +1,21 @@
+#### Project/deployment name ####
+# needs to be unique i.e. will be used as container and network name as well as for the vhost entry
+COMPOSE_PROJECT_NAME=rds-local
```

Comment:
please make sure to keep this when adding hte lastest version from the platform repo

Reply:
done.